src/hypofuzz/mutator.py

```diff
@@ -36,7 +36,21 @@ def _get_weights(self) -> list[float]:
         # This is related to the AFL-fast trick, but doesn't track the transition
         # probabilities - just node densities in the markov chain.
         weights = [
-            1 / min(self.corpus.behavior_counts[behavior] for behavior in fingerprint)
+            (
+                (
+                    1
+                    / min(
+                        self.corpus.behavior_counts[behavior]
+                        for behavior in fingerprint
+                    )
+                )
+                # it's possible for a fingerprint to be empty. If it is, we expect
+                # that to be the *only fingerprint in the corpus, so it doesn't
+                # matter what weight we assign to it (but we can't error by giving
+                # an empty iterable to `min`, or dividing by zero with `total`.)
+                if fingerprint
+                else 1
+            )
```

Comment:
I'd pull this out into an inner function; I think it'd be a lot clearer with statemnts vs these nested expressions.